Src/libCZI/CZIReader.cpp

```diff
@@ -96,6 +107,58 @@ CCZIReader::CCZIReader() : isOperational(false)
     return this->subBlkDir.GetPyramidStatistics();
 }
 
+/*virtual*/libCZI::IntPointAndFrameOfReference CCZIReader::TransformPoint(const libCZI::IntPointAndFrameOfReference& source_point, libCZI::CZIFrameOfReference destination_frame_of_reference)
```

Comment:
`destination_frame_of_reference` is value and not reference because we save 24bit (copying an 8bit int instead of creating a pointer to it), right?

Reply:
Yes, as rule [F.16](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rf-in) commands, I'd reckon.